Swiftfin.xcworkspace/contents.xcworkspacedata

```diff
@@ -2,9 +2,12 @@
 <Workspace
    version = "1.0">
    <FileRef
-      location = "group:JellyfinPlayer.xcodeproj">
+      location = "group:Swiftfin.xcodeproj">
    </FileRef>
    <FileRef
       location = "group:Pods/Pods.xcodeproj">
    </FileRef>
+   <FileRef
+      location = "group:Swiftfin.xcodeproj">
+   </FileRef>
```

Comment:
this looks to be a duplicate?

Reply:
Idk how Xcode handles it all as it builds just fine so it's okay